### PR TITLE
1.2.1 - Improve the Origin colours, specifically for users with colour deficiencies

### PR DESCRIPTION
**Pull request by paddyduke**
This branch makes several tweaks to the background and text colours in Origin to improve colour differentiation for users  who have difficulty with certain colour combinations.

The biggest changes are to the red tones, which are a lot less orange than previously. This makes them stand out a lot better against the green and amber than previously.

There's a screenshot below showing the differences but the red looks more pink here than it really is, so please check out the branch and take a look for yourself in Origin.

colours_before_and_after

### CODE REVIEW

File: docs/utilities/text-color.md [hunk 1]
```diff
@@ -274,23 +274,6 @@ New colors should be added only when they’re actually being used in a componen
...
-{% example scss %}
-.MyComponent {
-  @include text-color(fa-red--dark);
-}
-{% endexample %}
-</div>
-
-
-<div class="DocsExample DocsExample--grouped DocsExample--labelUtilityClasses">
-{% example html %}
```
Comment (Lindisaurus) [inline docs/utilities/text-color.md]: Why's this wee bit been deleted?

Comment (paddyduke) [inline docs/utilities/text-color.md]: See: https://github.com/fac/origin/commit/e417e8f2084b084520f16af7961edafd065056b2

Comment (Lindisaurus) [inline docs/utilities/text-color.md]: It's almost as if I could have taken the initiative to read the commit message. Got it! :smile:

**Comment by Lindisaurus** [comment]
Looks grand @paddyduke :+1:

**Pull request merged**